pagopa/io-app-design-system
# [SIW-965] Add `ModuleCredential` component

## Short description
This PR adds the `ModuleCredential` component

## List of changes proposed in this pull request
- Added `ModuleCredential` component
- Added examples

## How to test
Navigate to Modules > ModuleCredential, check that the component is rendere correctly, like in the preview screenshot.

## Preview

<img width="250" alt="Screenshot 2024-03-29 alle 11 52 05" src="https://github.com/pagopa/io-app-design-system/assets/6160324/5563fad6-8d40-4831-bf89-b0b2ac4e6135">
